_topics/using-queries-to-show-listings-for-a-specific-status

```diff
@@ -0,0 +1,20 @@
+  ---
```

Comment:
Everything on these four lines need to be aligned to the left:

```
  ---
  title
  layout
  ---
```

Needs to be:
```
---
title
layout
---
```